tests/test_plugin.py

```diff
@@ -574,6 +574,82 @@ def test_openchannel_hook(node_factory, bitcoind):
         l1.rpc.fundchannel(l2.info['id'], 100001)
 
 
+def test_openchannel_hook_error_handling(node_factory, bitcoind):
+    """ l2 uses a plugin that should fatal() crash the node.
+
+    This is because the plugin rejects a channel while
+    also setting a close_to address which isn't allowed.
+    """
+    opts = {'plugin': os.path.join(os.getcwd(), 'tests/plugins/openchannel_hook_accepter.py')}
```

Comment:
```suggestion
    opts = {'plugin': os.path.join(os.path.dirname(__file__), '..', 'tests/plugins/openchannel_hook_accepter.py')}
```
This is more stable and independent of the directory you run the tests from.

Reply:
didn't know that, though it looks ugly ;)